### PR TITLE
[improve-#13665] improve datax parser if sql contain *

### LINKED ISSUE #13665: [Improvement][Datax] if sql include ``` * ``` will print error information in log
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

image

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuchanghai**
## Purpose of the pull request
- #13665 
improve datax parser if sql contain *
this close : #13665 

## Brief change log

after change:

image

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Review by SbloodyS — DISMISSED** [review]
LGTM

File: dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java [hunk 1]
```diff
@@ -538,6 +543,11 @@ private String[] tryGrammaticalAnalysisSqlColumnNames(DbType dbType, String sql)
                             String.format("grammatical analysis sql column [ %s ] failed", item));
                 }
 
+                if (SELECT_ALL_CHARACTER.equals(item.toString())) {
+                    log.info("sql contain *,grammatical analysis sql failed");
```
Comment (Radeity) [inline dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java]:
```suggestion
                    log.info("sql contains *, grammatical analysis failed");
```

**Comment by rickchengx** [comment]
LGTM 👍

File: dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java (hunk 1 above)
Comment (fuchanghai) [reply dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java]: done

**Review by SbloodyS — APPROVED** [review]

**Pull request merged**